lib/cylc/network/https/base_client.py

```diff
@@ -106,6 +103,8 @@ def _get_data_from_url(self, url, json_data, method=None):
 
     def _get_data_from_url_with_requests(self, url, json_data, method=None):
         import requests
+        from urllib3.exceptions import InsecureRequestWarning
+        warnings.simplefilter("ignore", InsecureRequestWarning)
```

Comment:
So this doesn't always work for me, I played around with it on the python interpreter and iPython and got this:
```python
In [12]: import requests     
In [13]: import urllib3.exceptions

ImportError                               Traceback (most recent call last)
<ipython-input-13-6a522130414b> in <module>()
> 1 import urllib3.exceptions

ImportError: No module named urllib3.exceptions
```
This seems more reliable though:

```python
import requests
from requests.packages.urllib3.exceptions import InsecureRequestWarning
```